Test/git-issues/git-issue-2903.dfy

```diff
@@ -0,0 +1,25 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
```

Comment:
Could you use `%testDafnyForEachCompiler` here? The only thing you'd have to change in the test file is to avoid any unbounded `int` references, but that should be easy.

Reply:
From our chat offline - I ran into some errors here so it's not worth blocking on this; we'll need to convert a bunch of tests to do this later anyway. Resolving this for now.